code/datums/uplink_items/uplink_nuclear.dm

```diff
@@ -517,6 +509,14 @@
 
 // IMPLANTS
 
+/datum/uplink_item/implants/krav_implant
+	name = "Krav Maga Implant"
+	desc = "A biochip that teaches you Krav Maga when implanted, great as a cheap backup weapon. Warning: the biochip will override any other fighting styles such as CQC while active.
```

Comment:
```suggestion
	desc = "A biochip that teaches you Krav Maga when implanted, great as a cheap backup weapon. Warning: the biochip will override any other fighting styles such as CQC while active."
```

Reply:
Yup, noticed it when i looked back at VSC 💀